### PR TITLE
merge minisys back into system

### PR DESCRIPTION
**Pull request by Alizter**
This is on top of #15764

### CODE REVIEW

File: tools/coqdep/coqdep.ml [hunk 1]
```diff
@@ -8,34 +8,34 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
-(** The basic parts of coqdep are in [Coqdep_common]. *)
-module CD = Coqdep_common
+(** The basic parts of coqdep are in [Common]. *)
+open Coqdeplib.Common
```
Comment (ejgallego) [inline tools/coqdep/coqdep.ml:12]: In general I'm not in favor of using open like this, the global namespace is precious and we should avoid polluting it, specially when it was already prevented.

**Comment by ejgallego** [comment]
Ping @Alizter

File: tools/coqdep/coqdep.ml (hunk 1 above)
Comment (Alizter) [reply tools/coqdep/coqdep.ml]: That was from the other PR

**Review by ejgallego — APPROVED** [review]

File: tools/coqdep/coqdep.ml (hunk 1 above)
Comment (ejgallego) [reply tools/coqdep/coqdep.ml]: Yes I got confused by coqbot removing the needs rebase.

**Pull request merged**